interfaces/builtin/udisks2.go

```diff
@@ -352,6 +353,15 @@ KERNEL=="sr*", ENV{ID_VENDOR}=="SanDisk", ENV{ID_MODEL}=="Cruzer", ENV{ID_FS_LAB
 ENV{ID_PART_TABLE_TYPE}=="dos", ENV{ID_PART_ENTRY_TYPE}=="0x0", ENV{ID_PART_ENTRY_NUMBER}=="1", ENV{ID_FS_TYPE}=="iso9660|udf", ENV{UDISKS_IGNORE}="0"
 `
 
+const udisks2PermanentSlotUDevTag = `
+# This file contains udev rules for UDisks2 service.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="sr*|md*|fd*|sd*|mmcblk[0-9]|mspblk[0-9]", TAG+="%[1]s"
+SUBSYSTEMS=="usb", TAG+="%[1]s"
```

Comment:
I would be fine to remove the fd lines-- I was looking at the udev rules and saw it handled floppy a little differently, so put it. Really, it doesn't matter-- the apparmor policy currently only allows `/dev/sd*` and `/dev/mmcblk*` (and if you added it, `/dev/vd*`). I think you are right to just use block and usb subsystems and then we'll use apparmor globbing for which to allow.